app/views/about/show.html.erb

```diff
@@ -2,17 +2,12 @@
 
 <%= render partial: '/partials/hero/hero_about', locals: { title: I18n.t('about.hero') } %>
 
-<div class="about__header-secondary">
+<div class='about__header-secondary'>
   <h3><%= I18n.t('about.body') %></h3>
 </div>
 
-<div class="about__cards">
-  <div class="about__cards-card">
-    <%= I18n.t('about.footer.left') %>
-  </div>
-
-  <div class="about__cards-card">
-    <%= I18n.t('about.footer.right') %>
+<div class='about__cards'>
+  <div class='about__cards-card'>
+    <%= I18n.t('about.footer').html_safe %>
```

Comment:
This change seems to have made the left and right sides divide a little less cleanly

Before the split was consistently at the end of a sentence:

<img width="1344" alt="Screenshot 2022-03-28 at 18 04 37" src="https://user-images.githubusercontent.com/40244233/160450894-6bd5405a-f628-4580-bd45-7391f300bf86.png">

Now it's random based on screen width, and commonly mid-sentence:

<img width="1324" alt="Screenshot 2022-03-28 at 18 04 44" src="https://user-images.githubusercontent.com/40244233/160450932-75a380b0-bdbe-474a-93a4-575d3cc85995.png">

Reply:
moved back into two cards